packages/core/bootstrap/src/lib/external-adapter/validator.ts

```diff
@@ -19,6 +19,14 @@ import presetIncludes from './overrides/presetIncludes.json'
 import { Requester } from './requester'
 import { inputParameters } from './builder'
 
+const sharedInputConfig = excludableInternalAdapterRequestProperties.reduce<Record<string, false>>(
+  (config, name) => {
+    config[name] = false
```

Comment:
Could also make `sharedInputConfig` using the inputParameter syntax (gives it a type and may be more readable)